contrib/linux/tests/test_action_dig.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+
+# Copyright 2020 The StackStorm Developers
```

Comment:
Following the https://github.com/StackStorm/st2/pull/5003, the copyright string is the following:

```suggestion
# Copyright 2020 The StackStorm Authors.
```